arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -7,6 +7,7 @@
 
 #include "ffmpeg_init.h"
 #include "ffmpeg_video_input.h"
```

Comment:
This is not a change for this PR, but shouldn't `ffmpeg_video_input.h` be the first include since this is `ffmpeg_video_input.h`?

Reply:
Probably? The reason for this order was probably alphabetization, but I'm not 100% sure which takes priority in KWIVER style.